prov/gni/test/common.h

```diff
@@ -53,4 +54,9 @@ void calculate_time_difference(struct timeval *start, struct timeval *end,
 		int *secs_out, int *usec_out);
 int dump_cq_error(struct fid_cq *cq, void *context, uint64_t flags);
 
+static inline struct gnix_fid_ep *get_gnix_ep(struct fid_ep *fid_ep)
```

Comment:
Leave it.  Let's use language constructs over the preprocessor whenever possible.